web/packages/teleterm/src/ui/DocumentCluster/ActionButtons.tsx

```diff
@@ -180,3 +190,77 @@ async function getDatabaseUsers(appContext: IAppContext, dbUri: DatabaseUri) {
     throw e;
   }
 }
+
+function AppButton(props: {
+  isWebApp: boolean;
+  launchUrl: string;
+  connect(): void;
+  onLaunchUrl(): void;
+}) {
+  const ref = useRef<HTMLButtonElement>();
+  const [isOpen, setIsOpen] = useState(false);
+
+  if (!props.isWebApp) {
+    return (
+      <ButtonBorder size="small" onClick={props.connect}>
+        Set up connection
```

Comment:
```suggestion
        Connect
```

So that it matches other resources. "Set up connection" is going to look out of place, especially in list view.

<img width="901" alt="list-view" src="https://github.com/gravitational/teleport/assets/27113/9a0b2e5b-a160-445d-8f6b-695a81e2f8a8">